admin_guide/managing_projects.adoc

```diff
@@ -0,0 +1,169 @@
+that creates a project with the requested name, and assigns the requesting user
+to the "admin" role for that project.
+
+To create your own custom project template:
+
+. Start with the current default project template:
++
+----
+$ oadm create-bootstrap-project-template -o yaml > template.yaml
+----
+
+. Modify the template by adding objects or modifying existing objects, then load the template:
++
+----
+$ oc create -f template.yaml -n default
+----
+
+. Modify the *_master-config.yaml_* file to reference the loaded template:
++
+====
+----
+...
+projectConfig:
+  projectRequestTemplate: "default/project-request"
+  ...
+----
+====
+
+When a project request is submitted, the API substitutes the following parameters into the template:
+
+[cols="4,8",options="header"]
+|===
+|Parameter |Description
+
+|*PROJECT_NAME*
+|The name of the project. Required.
+
+|*PROJECT_DISPLAYNAME*
+|The display name of the project. May be empty.
+
+|*PROJECT_DESCRIPTION*
+|The description of the project. May be empty.
+
+|*PROJECT_ADMIN_USER*
+|The username of the administrating user.
+
+|*PROJECT_REQUESTING_USER*
+|The username of the requesting user.
+|===
+
+Access to the API is granted to developers with the
+link:../architecture/additional_concepts/authorization.html#roles[`self-provisioner`
+role] and the `self-provisioners` cluster role binding. This role is available
+to all authenticated developers by default.
+
+=== Disabling Self-provisioning
+Deleting the `self-provisioners`
+link:../architecture/additional_concepts/authorization.html#roles[cluster role
+binding] will deny permissions for self-provisioning any new projects. When
+disabling self-provisioning, set the `projectRequestMessage` parameter in the
+*_master-config.yaml_* file instructing developers on how to request a new
+project. This parameter is a string that will be presented to the developer in
+the web console and command line when they attempt to self-provision a project.
+For example:
+
+----
+Contact your system administrator at projectname@example.com to request a project.
+----
+
+or:
+
+----
+To request a new project, fill out the project request form located at
+https://internal.example.com/openshift-project-request.
+----
+
+[[using-node-selectors]]
+== Using Node Selectors
+
+Node selectors are used in conjunction with labeled nodes to control pod placement. 
+
+[NOTE]
+====
+Labels can be assigned link:../install_config/install/advanced_install.html#configuring-node-host-labels[during an advanced installation], or link:../admin_guide/manage_nodes.html#updating-labels-on-nodes[added to a node after installation].
+====
+
+As an OpenShift cluster administrator, you can set the cluster-wide default node selector to restrict pod placement to specific nodes. For example, if you have an OpenShift topology with multiple regions, node selectors help restrict specific OpenShift projects to only deploy pods onto nodes in a specific region.
```

Comment:
@sosiouxme I moved the "Alternatively..." section up, but from what I understand, this is the exact same process, just a different way to go about it. If that's not true, then I've definitely received conflicting information from somewhere. 

Either way, now the example use case I was putting forward doesn't exist anymore, and this is just a topic full of info...